tcpb/utils.py

```diff
@@ -44,10 +48,52 @@ def atomic_input_to_job_input(atomic_input: AtomicInput) -> pb.JobInput:
         pb.JobInput.ImdOrbitalType,
         atomic_input.keywords.pop("imd_orbital_type", "NO_ORBITAL").upper(),
     )
-
-    # Drop keyword terms already applied to Molecule
-    atomic_input.keywords.pop("charge", None)
-    atomic_input.keywords.pop("spinmult", None)
+    
+    # Second geometry for ci_vec_overlap job
+    # TODO: Not tested yet!
+    try:
+        geom2 = atomic_input.keywords.pop("geom2")
+        if isinstance(geom2, ndarray):
+            geom2 = geom2.flatten()
+        if len(atomic_input.molecule.symbols) != len(geom2) / 3.0:
+            raise ValueError(
+                "Geometry provided to geom2 does not match atom list"
+            )
+        del ji.xyz2[:]
+        ji.xyz2.extend(geom2)
+    except KeyError:
+        pass
```

Comment:
Is there a way we can nest this logic under some sort of `if ci_vec_overlap_job is True` sort of statement? That way we only execute this logic when explicitly declared by the caller instead of implicitly when certain values get passed? That can help avoid issues where accidentally passed values create weird bugs that are implicit and hard to track down :)